### DISCUSSION TITLE
gqlgen generate broken names for methods

### DISCUSSION THREAD

**Discussion by droslean** [99designs/gqlgen#1980] [Q&A]
### What happened?

`return ec.marshalOEnumApp2ᚖgithubᚗcomᚋdrosleanᚋcitadelleᚋpkgᚋgraphᚋmodelᚐEnumApp(ctx, field.Selections, res)`

### What did you expect?

To generate a better name for that method

### versions
 - `gqlgen version`?  `v0.14.0-dev`
 - `go version`? `go version go1.17.1 linux/amd64`
 - dep or go modules? go modules

### How to replicate? Just follow the official docs.

```console
$ go mod init
$ go get github.com/99designs/gqlgen
$ go run github.com/99designs/gqlgen init --verbose
```

And it will generate broken names in `graph/generated/generated.go`

**Comment by droslean** [comment]
Also, the version is incorrect since I am using `github.com/99designs/gqlgen v0.16.0` but the version flag says `v0.14.0-dev`

**Comment by BenjamenMeyer** [comment]
Not everyone uses an auto-completion IDE so adding unicode/extended-ASCII characters into generated names is not very user friendly since they're not easy/possible to type, in addition to being hard to read.

**Comment by BenjamenMeyer** [comment]
It would be better to use CamelCase or The_Underscore_Method. It would be great if this was configurable.
